### PR TITLE
Add more information on how to configure Gradle's native task

### PR DESCRIPTION
**Pull request by geoand**
See discussion at: https://groups.google.com/forum/#!topic/quarkus-dev/XWG6OvFxO9s

### CODE REVIEW

File: docs/src/main/asciidoc/gradle-tooling.adoc [hunk 1]
```diff
@@ -333,6 +344,22 @@ The produced executable will be a 64 bit Linux executable, so depending on your
 it may no longer be runnable.
 However, it's not an issue as we are going to copy it to a Docker container.
 
+Another way of customizing the native build image build process is to define a custom task. If for example it is required to set the `enableHttpUrlHandler`, it can be done like so:
+
+[source,groovy]
+----
+task customBuildNative(type: io.quarkus.gradle.tasks.QuarkusNative) {
+    enableHttpUrlHandler = true
+}
```
Comment (gsmet) [inline docs/src/main/asciidoc/gradle-tooling.adoc]: You can't configure an existing task?

Comment (geoand) [reply docs/src/main/asciidoc/gradle-tooling.adoc]: Nope... I tried it and gradle failed. Unless I did it wrong :)

Comment (geoand) [reply docs/src/main/asciidoc/gradle-tooling.adoc]: Hm, maybe there is some other trick

Comment (geoand) [reply docs/src/main/asciidoc/gradle-tooling.adoc]: I think I got it to work. I'll update the docs momentarily. Thanks for nudging me in the right direction :P

**Comment by geoand** [comment]
@gsmet PR updated with the better solution

**Comment by geoand** [comment]
Probably a candidate for `0.22` as well

**Review by gsmet — APPROVED** [review]

**Pull request merged**